### DISCUSSION TITLE
How to Create a Factorial Simulation for Different planting density

### DISCUSSION THREAD

**Discussion by Malitoz** [APSIMInitiative/ApsimX#9178] [General]
I wish to run a factorial simulation with 3 properties as planting density (4, 7 and 10).  How do I set this up?
![image_2024-07-19_172234655](https://github.com/user-attachments/assets/249f844b-d655-4090-a8aa-729656ce1d31)

**Comment by Malitoz** [comment]
[SowingRule].Script.SowDate = dd-mmm

**Reply by i-heis** [reply]
Could you share your script for the PD2 CompositeFactor node? I would like to reconstruct it and it does not work with the one line that you posted, thanks :)

**Reply by Malitoz** [reply]
Hello @i-heis , I changed the structure for the factorial. 
![image_2024-08-01_190130982](https://github.com/user-attachments/assets/2433ba69-5df5-4ce0-9893-b5ed4e3aeb67)

**Comment by Malitoz** [comment]
Here's the whole simulation Script, I removed the CompositeFactors for the planting densities and replaced it with set range for the FertiliserRule.
________________________________
